ci-operator/config/opendatahub-io/model-registry/opendatahub-io-model-registry-main.yaml

```diff
@@ -38,6 +38,48 @@ tests:
         requests:
           cpu: 100m
     workflow: hypershift-hostedcluster-workflow
+- as: e2e-odh-model-registry-standalone
+  steps:
+    cluster_profile: aws-opendatahub
+    env:
+      BASE_DOMAIN: openshift-ci-aws.rhaiseng.com
+      HYPERSHIFT_AWS_REGION: us-west-2
+      HYPERSHIFT_NODE_COUNT: "3"
+    test:
+      - as: main
+        cli: latest
+        commands: |
+          # Print last commit of current code
+          git log --name-status HEAD^..HEAD
+          # Print image used for tests
+          echo "Using image ${ODH_MODEL_REGISTRY_IMAGE}"
+          # Install Robot Framework tests requirements
+          pip3.9 install -r test/robot/requirements.txt
+          # Install Python client for model-registry
+          pip3.9 install -e clients/python
+          # Change URL for tests
+          sed -i -e "s/localhost/model-registry-service/" test/robot/*.resource test/robot/*.py
+          # Change image of model-registry deployment
+          sed -i -e "s|image: quay.io/opendatahub/model-registry.*|image: ${ODH_MODEL_REGISTRY_IMAGE}|" manifests/kustomize/base/model-registry-deployment.yaml
+          # Create namespace for tests
+          oc create namespace kubeflow
+          # Deploy model-registry instance
+          oc apply -k manifests/kustomize/overlays/postgres
```

Comment:
I'd rather opt for the mysql/mariadb one, given recent discussions

Reply:
I would like to test all variants later too 👍